integration_test/cloudbuild-integration-test.yaml

```diff
@@ -111,7 +205,9 @@
   - "--location=${_LOCATION}"  
   - "--quiet"
   waitFor:
-  - diff-dcm
+  - check-store-curl
+  - run-find-scu
+  - run-move-scu
```

Comment:
Let's add run-commitment-scu here for consistency?

Reply:
Ok